components/Layout.js

```diff
@@ -2,28 +2,90 @@ import Link from 'next/link';
 
 export const Layout = ({ children }) => (
   <>
-    <header className="w-full h-110 bg-stone-300">
-      <div className="w-full bg-light-50 p-2 md:p-5 grid grid-cols-4">
-        <Link href="/">
-          <a href="home">Hiswiki</a>
-        </Link>
+    <header className="w-full h-[440px] bg-[url('../styles/img/header_background.jpg')] bg-no-repeat bg-cover bg-stone-300">
+      <div className="w-full  bg-light-50">
+        <div className="max-w-screen-xl mx-auto p-2 md:p-5 flex">
+          <Link href="/">
+            <a
+              className="flex-initial font-yrsa font-bold uppercase text-4xl pr-8"
+              href="home"
+            >
+              Hiswiki
+            </a>
+          </Link>
+          <div class="flex-initial">
+            <form>
+              <label className="sr-only" for="search">
```

Comment:
What is `sr-only` about?

Reply:
visible only for screen readers https://tailwindcss.com/docs/screen-readers